apps/extension/src/background/keyring/types.ts

```diff
@@ -0,0 +1,32 @@
+export type Bip44Path = {
+  account: number;
+  change: number;
+  index: number;
+};
+
+export type MnemonicState = {
+  id: string;
+  alias?: string;
+  phrase: Uint8Array;
+};
+
+export type DerivedAccount = {
+  id: string;
+  address: string;
+  alias?: string;
+  bip44Path: Bip44Path;
```

Comment:
nit: listing optionals last
```suggestion
  bip44Path: Bip44Path;
  alias?: string;
```